runtime/interpreter/encode.go

```diff
@@ -225,6 +225,7 @@ const (
 	CBORTagEntitlementMapStaticAuthorization
 	CBORTagEntitlementSetStaticAuthorization
 	CBORTagInclusiveRangeStaticType
+	CBORTagInaccessibleStaticAuthorization
```

Comment:
We might want to clean this up while we still can: Given there is no on-chain state stored yet for `InclusiveRange`, we can change the tag, separating it from the authorization kinds, and leaving some space for future additions and changes in this "group" of tags:

```suggestion
	CBORTagInaccessibleStaticAuthorization
	_
	_
	_
	_
	CBORTagInclusiveRangeStaticType
```

I arbitrarily chose 4, but we can use fewer or more